components/device-mgt/org.wso2.carbon.device.mgt.core/src/main/java/org/wso2/carbon/device/mgt/core/operation/mgt/dao/impl/CommandOperationDAOImpl.java

```diff
@@ -134,7 +134,8 @@ public CommandOperation getOperation(int id) throws OperationManagementDAOExcept
             while (rs.next()) {
                 commandOperation = new CommandOperation();
                 commandOperation.setId(rs.getInt("ID"));
-                commandOperation.setEnabled(rs.getInt("ENABLED") != 0);
+                //commandOperation.setEnabled(rs.getInt("ENABLED") != 0);
```

Comment:
Remove commented line.

Reply:
Fixed in https://github.com/wso2/carbon-device-mgt/pull/1275